editor/js/Menubar.Edit.js

```diff
@@ -195,7 +195,7 @@ Menubar.Edit = function ( editor ) {
 				" material(s) were changed.\n" + errors.join( "\n" ) );
 
 	} );
-	options.add( option );
+//	options.add( option );
```

Comment:
Is there a way to pass a config and then enable or disable these items based on it? That way we can put up a PR to original repo.